Relativity.CustomPages.md

```diff
@@ -0,0 +1,17 @@
+# Relativity.CustomPages Version Compatibility
+
+![Nuget](https://img.shields.io/nuget/v/Relativity.CustomPages)

```

Comment:
Mistake on my sample can you change this to - 
[![nuget](https://img.shields.io/nuget/v/Relativity.CustomPages.svg)](https://www.nuget.org/packages/Relativity.CustomPages)